.azure-pipelines/js-mgmt-pr.yml

```diff
@@ -1,15 +1,30 @@
-# Node.js
-# Build a general Node.js project with npm.
-# Add steps that analyze code, save build artifacts, deploy, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
-trigger:
-  - master
+pr:
```

Comment:
What is the dfference between ci and pr for your yml files?

Reply:
Our general convention is to have one pipeline with a "ci" tag for PR validation and then using the same yaml file but perhaps with some addition jobs that are conditioned as the internal full build.

Reply:
pr is the build that runs on every PR created, including the checks for version bump. ci runs on master build and produces drops.

Reply:
@mitchdenny could you please work with @kpajdzik to try and make these builds more consistent with what we are planning? Ideally we would try and combine builds so we can do scoped builds like we are doing in other languages.